src/pages/actualites/index.page.test.tsx

```diff
@@ -0,0 +1,208 @@
+
+			expect(result).toEqual({ notFound: true, revalidate: 1 });
+		});
+
+		describe('lorsque la récupération des actualités est en succès', () => {
+			it('transmet les props', async () => {
+				jest.spyOn(dependencies.actualitesDependencies.consulterActualitesUseCase, 'handle').mockResolvedValue(createSuccess([anActualite()]));
+
+				const result = await getStaticProps();
+
+				expect(result).toEqual(expect.objectContaining({
+					props: {
+						cartesActualites: [anActualite()],
+					},
+				}));
+			});
+
+			it('envoie les analytics de la page à son affichage', () => {
+				const carteActualites = anActualiteList();
+				const analyticsService = aManualAnalyticsService();
+
+				render(
+					<DependenciesProvider analyticsService={analyticsService}>
+						<ActualitesPage cartesActualites={carteActualites}/>
+					</DependenciesProvider>,
+				);
+
+				expect(analyticsService.envoyerAnalyticsPageVue).toHaveBeenCalledWith({
+					page_template: 'contenu_liste_niv_1',
+					pagegroup: 'service_jeune_liste',
+					pagelabel: 'contenu_liste_niv_1',
+					'segment-site': 'contenu_liste',
+				});
+			});
+
+			it('doit rendre du HTML respectant la specification', () => {
+				const carteActualites = [anActualite({ titre: 'Actualité 1' }), anActualite({ titre: 'Actualité 2' }), anActualite({ titre: 'Actualité 3' })];
+
+				const { container } = render(<DependenciesProvider analyticsService={aManualAnalyticsService()}>
+					<ActualitesPage cartesActualites={carteActualites}/>
+				</DependenciesProvider>);
+
+				expect(container.outerHTML).toHTMLValidate();
+			});
+
+			it('n‘a pas de défaut d‘accessibilité', async () => {
+				const carteActualites = [anActualite({ titre: 'Actualité 1' }), anActualite({ titre: 'Actualité 2' }), anActualite({ titre: 'Actualité 3' })];
+
+				const { container } = render(
+					<DependenciesProvider analyticsService={aManualAnalyticsService()}>
+						<ActualitesPage cartesActualites={carteActualites}/>
+					</DependenciesProvider>);
+
+				await expect(container).toBeAccessible();
+			});
+
+			it('affiche la liste des actualités', () => {
+				const carteActualites = [
+					anActualite({ extraitContenu: 'Je suis un extrait', titre: 'Actualité 1' }),
+					anActualite({ titre: 'Actualité 2' }),
+					anActualite({ titre: 'Actualité 3' }),
+				];
+				render(
+					<DependenciesProvider analyticsService={aManualAnalyticsService()}>
+						<ActualitesPage cartesActualites={carteActualites}/>
+					</DependenciesProvider>,
+				);
+
+				expect(screen.getByRole('list')).toBeVisible();
+				const actualitesList = screen.getAllByRole('listitem');
+				expect(actualitesList).toHaveLength(3);
+
+				const premiereActualite = actualitesList[0];
+				expect(within(premiereActualite).getByRole('heading', { level: 2, name: 'Actualité 1' })).toBeVisible();
+				expect(within(premiereActualite).getByText('Je suis un extrait')).toBeVisible();
+			});
+
+			describe('lien de redirection de l‘actualité', () => {
```

Comment:
Un exemple de disjonction de cas qui est bien géré. Au lieu d'avoir deux describe succès / échec. T'as un describe global et l'ensemble des it au même niveau d'indentation qui traite tous les cas possibles. Les deux façons de faire me vont

Reply:
Autre commentaire, ce cas là n'est pas déjà testé dans le composant ? C'est une propriété qui est testée en composant dans les FlippingCard (MesuresJeunes / Employeurs)

Reply:
Je test que le wording du lien donnée à Flipping card est le bon